_includes/footer.html

```diff
@@ -1,154 +1,47 @@
       </p>
       <div class="grid-row footer-contact-links">
-        <div class="tablet:grid-col">
-          <div class="usa-media-block">
-            <div class="usa-media-block__img circle-5 bg-accent-cool display-flex flex-row flex-align-center flex-justify-center text-white">
-              <svg class="usa-icon usa-icon--size-3" aria-hidden="true" role="img" focusable="false">
-                <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#help"></use>
-              </svg>
-            </div>
-            <div class="usa-media-block__body">
-              <h3>Have an idea or an issue?</h3>
-              <a href="{{ site.repos[0].url }}/issues/new/choose">
-                Reach out in GitHub
-              </a>
-            </div>
-          </div>
-        </div>
-        <div class="tablet:grid-col">
-          <div class="usa-media-block">
-            <div class="usa-media-block__img circle-5 bg-accent-cool display-flex flex-row flex-align-center flex-justify-center text-white">
-              <svg class="usa-icon usa-icon--size-3" aria-hidden="true" role="img" focusable="false">
-                <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#people"></use>
-              </svg>
-            </div>
-            <div class="usa-media-block__body">
-              <h3>Engage with the community</h3>
-              <a href="https://designsystem.digital.gov/about/community/">
-                Find a channel that works for you
-              </a>
-            </div>
-          </div>
-        </div>
-        <div class="tablet:grid-col">
-          <div class="usa-media-block">
-            <div class="usa-media-block__img circle-5 bg-accent-cool display-flex flex-row flex-align-center flex-justify-center text-white">
-              <svg class="usa-icon usa-icon--size-3" aria-hidden="true" role="img" focusable="false">
-                <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#mail"></use>
-              </svg>
-            </div>
-            <div class="usa-media-block__body">
-              <h3>Subscribe to our newsletter</h3>
-              <a class="usa-link usa-link--external" href="https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?qsp=GSA_TTS">
-                Sign up for USWDS updates
-              </a>
+        {% for callout in footer.callouts %}
+          <div class="tablet:grid-col">
+            <div class="usa-media-block">
+              <div class="usa-media-block__img circle-5 bg-accent-cool display-flex flex-row flex-align-center flex-justify-center text-white">
+                <svg class="usa-icon usa-icon--size-3" aria-hidden="true" role="img" focusable="false">
+                  <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#{{ callout.icon }}"></use>
+                </svg>
+              </div>
+              <div class="usa-media-block__body">
+                <h3>{{ callout.tagline }}</h3>
+                <a href="{{ callout.link_href }}">
```

Comment:
Should these have at least the `usa-link` class?

Reply:
@amyleadem normally I would say yes, but other site links are styled globally. I didn't want to go against that pattern.